### PR TITLE
Removes staff powers from maintainers in discord

### PR DESCRIPTION
**Pull request by LoliconSlayer**
image

### CODE REVIEW

**Comment by N3D6** [comment]
Lol based

**Comment by LoliconSlayer** [comment]
Also conflicts with #302

**Comment by TheGamerdk** [comment]
Removes reboot and mhelp stuff.
Maybe toggle ooc shouldn't be removed either

**Comment by TheGamerdk** [comment]
also breaks senior admin and head dev rank

**Comment by TheGamerdk** [comment]
and council and host

**Review by adamsong — CHANGES_REQUESTED** [review]
SA still inherits maint

**Review by JamieD1 — CHANGES_REQUESTED** [review]
Imagine trying to change perms with a PR

**Comment by LoliconSlayer** [comment]
Good points, wait one so I can redo all of the perms

**Review by adamsong — APPROVED** [review]
Good, puts maintainers in their place, just above mentors

**Comment by Anvilman6** [comment]
Good idea but
code is evil and shifting a building block will topple the tower

File: data/discord.json [hunk 1]
```diff
@@ -13,20 +13,20 @@
...
       "permissions": ["userverify"]
     },
     "retmin": {
+      "inherits": "maintainer",
+      "permissions": ["tempban", "ban", "unban", "kick", "ticket", "whitelist", "kickself", "note", "staffban", "listadmins"]
+    },
+    "maintainer": {
       "inherits": "mentor",
-      "permissions": ["tempban", "ban", "unban", "kick", "reboot", "ticket", "whitelist", "kickself", "toggleooc", "note", "staffban", "listadmins"]
+      "permissions": ["reboot", "toggleooc"]
```
Comment (TheGamerdk) [inline data/discord.json:29]:
```suggestion
      "permissions": ["reboot", "toggleooc", "kickself"]
```

File: data/discord.json [hunk 2]
```diff
@@ -13,20 +13,20 @@
...
-      "permissions": []
+      "permissions": ["addao", "removeao", "addmentor", "removementor"]
     },
     "staff": {
       "inherits": "retmin",
       "permissions": ["userverify"]
     },
     "retmin": {
+      "inherits": "maintainer",
+      "permissions": ["tempban", "ban", "unban", "kick", "ticket", "whitelist", "kickself", "note", "staffban", "listadmins"]
```
Comment (TheGamerdk) [inline data/discord.json:25]:
```suggestion
      "permissions": ["tempban", "ban", "unban", "kick", "ticket", "whitelist", "note", "staffban", "listadmins"]
```

**Review by TheGamerdk — CHANGES_REQUESTED** [review]

File: data/discord.json (hunk 1 above)
Comment (adamsong) [reply data/discord.json:29]: Power creep

**Comment by LoliconSlayer** [comment]
@TheGamerdk The kickself issues will be addressed by #302 assuming this gets merged first

**Comment by TheGamerdk** [comment]
> @TheGamerdk The kickself issues will be addressed by #302 assuming this gets merged first

doesn't matter

**Comment by LoliconSlayer** [comment]
Alrighty then

**Comment by TheGamerdk** [comment]
> Alrighty then

Thanks <3, best wiki

**Review by TheGamerdk — APPROVED** [review]

**Review by JamieD1 — CHANGES_REQUESTED** [review]
Will need to be done on the box.

**Comment by JamieD1** [comment]
Fucking hate you bibby. More then Bovril.

**Pull request merged**